resources/dicts/patrols/general/training.json

```diff
@@ -11690,5 +11690,371 @@
+           },
+           {
+               "text": "s_c confesses {PRONOUN/s_c/poss} fear that one day, all of c_n will be in danger, and only {PRONOUN/s_c/subject} will be able to save them. p_l comforts s_c and assures {PRONOUN/s_c/object} that c_n will stand together against any threat. Feeling reassured, s_c asks if p_l will teach {PRONOUN/s_c/object} some moves anyway - for fun.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "insecure", "nervous", "gloomy" ],
+               "relationships": [
+                   {
+                       "cats_to": ["p_l"],
+                       "cats_from": ["app1"],
+           "mutual": true,
+                       "values": ["platonic", "comfort", "trust", "respect"],
+                       "amount": 15
+                   } ],
+               "weight": 20
+           },
+           {
+               "text": "s_c's passion surprises p_l when {PRONOUN/s_c/subject} {VERB/s_c/declare/declares} that {PRONOUN/s_c/subject} {VERB/s_c/want/wants} to be able to defend those that cannot defend themselves - kits, elders, every cat in c_n, and even those beyond their borders. p_l agrees to show s_c some moves.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "righteous", "faithful", "thoughtful", "sincere" ],
+               "relationships": [
+                   {
+                       "cats_to": ["p_l"],
+                       "cats_from": ["app1"],
+           "mutual": true,
+                       "values": ["platonic", "comfort", "trust", "respect"],
+                       "amount": 15
+                   } ],
+               "weight": 20
+           },
+           {
+               "text": "s_c explains that doing good and upholding the code is always going to be a struggle against evil and rule-breakers. {PRONOUN/s_c/subject/CAP} {VERB/app1/need/needs} to be powerful to enforce the code. p_l agrees to teach {PRONOUN/s_c/object} some moves.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "righteous", "strict" ],
+               "relationships": [
+                   {
+                       "cats_to": ["p_l"],
+                       "cats_from": ["app1"],
+           "mutual": true,
+                       "values": ["platonic", "comfort", "trust", "respect"],
+                       "amount": 15
+                   } ],
+               "weight": 20
+           },
+           {
+               "text": "s_c describes a future where {PRONOUN/s_c/subject} might be alone in the territory, or even beyond the border, and might need to take care of {PRONOUN/s_c/self}. p_l reassures s_c that this is unlikely to happen to {PRONOUN/s_c/object} any time soon, but agrees to show {PRONOUN/s_c/object} some moves.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "adventurous" ],
```

Comment:
perhaps "lonesome" would also be fitting here? thinking about a future where they're alone.  maybe also "nervous"? a future where they have no one to rely on?